src/main/java/frc/robot/commands/autodrive/AutoAlignWithAmp.java

```diff
@@ -23,24 +23,24 @@ public class AutoAlignWithAmp extends Command {
 
   private final ProfiledPIDController turnController =
       new ProfiledPIDController(
-          ShooterConstants.AUTO_LINEUP_ROTATION_P,
-          ShooterConstants.AUTO_LINEUP_ROTATION_I,
-          ShooterConstants.AUTO_LINEUP_ROTATION_D,
-          ShooterConstants.AUTO_LINEUP_ROTATION_CONSTRAINTS);
+          FlywheelConstants.AUTO_LINEUP_ROTATION_P,
```

Comment:
why did we rename this? I think `ShooterConstants` was more accurate, especially for this case. The auto lineup stuff doesn't have anything to do with specifically the flywheel

Reply:
it was renamed to match the subsystem name. imo auto lineup constants should actually go in trajectory constants bc the shooter isn't the thing thats lining up w/ the amp, the swerve is.

Reply:
yeah we should move it to trajectory constants, also consider renaming trajectory constants to auto constants

Reply:
Looks like im right so Im keeping it as flywheel

Reply:
What

Reply:
Bro did not read the conversation

Reply:
Bro did not read the conversation